builder/data.py

```diff
@@ -345,6 +345,8 @@ class PKG_TOOLS(Enum):
 
         'imports': ['llvm'],
 
+        'compiler_packages': ['libstdc++-9-dev'],
```

Comment:
This should probably be `apt_compiler_packages` since this package name will probably only work on debian/ubuntu